calico-cloud/image-assurance/exclude-vulnerabilities-from-scan-results.mdx

```diff
@@ -0,0 +1,81 @@
+1. Select an image from the list.
+1. On the scan results panel that appears, expand a package, and then click **Actions > Add Exception**.
+1. On the **Add exception** dialog, select a scope, enter an explanation for why you're creating an exception, and then click **Save**.
+
+## Exclude multiple vulnerabilities at the same time
+
+To exclude large numbers of vulnerabilities, you can organize the required information as a CSV file and import it directly to Calico Cloud.
+
+1. Optional: You can start with a preformatted list of all vulnerabilities in your cluster, and then edit the list to include only those vulnerabilities you want to exclude.
+   1. From the Calico Cloud Manager UI, go to **Image Assurance > All Scan Results**.
+   1. On the **All Scan Results** page, click the **Export** button, and then click **Export data**.
+   1. On the **Export** dialog that appears, select the **CSV** data type and the **Export a row for each image and CVE IDs** option for the table style.
+      Click **Export** to download the CSV file.
+   1. Edit the CSV file
+
+1. Prepare the CSV file with information about the vulnerabilities you want to exclude.
+   The file must contain the following information:
+
+   | Column header | Description | Example |
+   | -- | -- | -- |
+   | `CVE` | The CVE identifier. Required by the `any`, `image`, and `image_tag` scopes. | `CVE-2024-1234` |
+   | `Registry`| The URL path to the container registry. Required by the `image` and `image_tag` scopes. | `mycontainerregistry.io/my-org` |
+   | `Repository`| The container image name. Required by the `image` and `image_tag` scopes.| `my-application` |
+   | `Tags`| A JSON list of image tags. Required by the `image_tag` scope. | `"[""v1.2.3"",""v2.3.4""]"` |
+   | `Justification` | Your explanation for why you're excluding this vulnerability. Required by the `any`, `image`, and `image_tag` scopes.| `This one is a false positive.` |
+   | `Scope` | Determines whether the vulnerability exception applies only to specific tagged images, to any image in a specific repository, or to any image where the vulnerability is found. One of the following values is required:<br/>&bull; `any`: The exception applies to all images.<br/>&bull; `image`: The exception applies to all versions of the image<br/>&bull; `image_tag`: The exception applies to specific tags of an image | `image_tag` |
+
+   :::tip
+   An exported vulnerability list (see step 1) includes many more columns than what is required to import vulnerability exceptions in bulk.
+   You do not need to remove or reorganize the extraneous information, but you do need to add two new columns for `Justification` and `Scope`.
+   :::
+
+   Example: a CSV list of vulnerability exclusion definitions
+
+   ``` title="example-vulnerability-exclusions.csv"
+   CVE,Registry,Repository,Tags,Justification,Scope
+   CVE-2024-1234,mycontainerregistry.io/my-org,my-application,"[""v3.4.5""]",justification,image_tags
```

Comment:
The last value in this image_tags needs to be change from image to **image**.

Reply:
Changed.